crates/bitwarden/src/sdk/request/projects_request.rs

```diff
@@ -9,9 +9,36 @@ pub struct ProjectGetRequest {
     pub id: Uuid,
 }
 
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct ProjectCreateRequest {
+    /// Organization where the project will be created
+    pub organization_id: Uuid,
+
+    pub name: String,
+}
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct ProjectPutRequest {
```

Comment:
Should this be called updateRequest? `Put` feels like an implementation detail.

Reply:
This was to follow the convention in [secrets_request.rs](https://github.com/bitwarden/sdk/blob/master/crates/bitwarden/src/sdk/request/secrets_request.rs) which uses `SecretPutRequest`. We definitely can refactor out `Put` in multiple locations and use `Update` though. This could be done in a different ticket, this one, or we take a different approach?

Reply:
Discussed this, we will address this in a separate PR